clustoapi/util.py

```diff
@@ -106,6 +106,9 @@ def expanded():
         if isinstance(obj, clusto.drivers.resourcemanagers.ResourceManager):
             result['count'] = obj.count
 
+        if obj.entity.type == 'server':
```

Comment:
IPs are not restricted to servers, switches and some other devices have that too. You should probably change this to either a try/catch or a 'if get_ips in dir(obj)' conditional, whatever is less expensive.